feelpp/pyfeelpp/feelpp/discr.cpp

```diff
@@ -214,8 +214,9 @@ PYBIND11_MODULE(_discr, m )
     defDiscr<Pch_type<Mesh<Simplex<1,1,1>,double,0,uint32_type>,1,double,PointSetFekete,0>>( m );
     defDiscr<Pch_type<Mesh<Simplex<1,1,1>,double,0,uint32_type>,2,double,PointSetFekete,0>>( m );
     defDiscr<Pch_type<Mesh<Simplex<1,1,1>,double,0,uint32_type>,3,double,PointSetFekete,0>>( m );
-    
-    defDiscr<Pch_type<Mesh<Simplex<2,1,2>,double,0,uint32_type>,1,double,PointSetFekete,0>>( m );
+
+    defDiscr<FunctionSpace<Mesh<Simplex<2>>, bases<Lagrange<1, Scalar, Continuous, PointSetFekete>>>>(m);
+    // defDiscr<Pch_type<Mesh<Simplex<2,1,2>,double,0,uint32_type>,1,double,PointSetFekete,0>>( m );
```

Comment:
I had to do this to avoid the type deduction. Should we keep the two types (with and without PointSetFekete), and change the names, or keep only one @prudhomm ?

Reply:
this behavior will change: but keep both for now

Reply:
Ok but if we keep all, with scalar, vectorial, continuous, discontinuous, fekete or not, with 1D to 3D, it makes 48 functional spaces to instanciate. Is it ok ?